sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClient.java

```diff
@@ -493,7 +493,7 @@ public ConfigurationSetting deleteSetting(ConfigurationSetting setting) {
      * {@codesnippet com.azure.data.applicationconfig.configurationclient.listSettingRevisions#settingSelector}
      *
      * @param selector Optional. Used to filter configuration setting revisions from the service.
-     * @return {@link PagedIterable} of the {@link ConfigurationSetting}
+     * @return Revisions of the ConfigurationSetting
```

Comment:
Why remove the `{@link ConfigurationSetting}`? If you refactor this class, it'll pick up the link and rename it.

Reply:
added  it  back, I was trying to solve @link issue but it was because of something else.